pipelines.md

```diff
@@ -81,17 +83,73 @@ The default tasks and pipelines can be updated by forking the Kabanero Pipelines
 
 ### Tasks
 
-- [build-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/build-task.yaml)
+- [validate-active-stack-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/validate-active-stack-task.yaml)
+
+This task will validate the stack is allowed to build an deploy on the cluster.  It checks the digest of the stack image specified in the `.appsody-config.yaml` of the project and validates it matches the digest of the stack image that's active on the collection.  If they do not match, the pipeline will fail and none of the other steps will execute.
 
-   This file builds a container image from the artifacts in the git-source repository by using [Buildah](https://github.com/containers/buildah). After the image is built, the `build-task` file publishes it to the docker-image URL by using Buildah.
+- [build-push-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/build-push-task.yaml)
+
+   This file builds a container image from the artifacts in the git-source repository by using `appsody build`.  The appsody build command will leverage the [Buildah](https://github.com/containers/buildah) options. After the image is built, the image will get published to the container registry that is configured. The build-push-task will also generate the `app-deploy.yaml` that is used by the `deploy-task`.  If there is already a copy of the `app-deploy.yaml` file in the source repository, it will get merged with the new one genrated by this step.
```

Comment:
Change
will leverage
to
leverages

Change
the image will get published
to
the image is published

Change
task will also generate
to
task also generates

Change
it will get merged
to
it is merged

Change
genrated
to
generated